aip/0123.md

```diff
@@ -72,6 +72,60 @@ message Topic {
 - Patterns **must** correspond to the [resource name][aip-122].
 - Pattern variables (the segments within braces) **must** use `snake_case`, and
   **must not** use an `_id` suffix.
+- Pattern variables (the segments within braces) **must** begin with a
```

Comment:
One of these two bullets must be wrong. (I think the bullet on line 73 is correct and this one is wrong.)

Reply:
I don't follow that. The bullets could be combines, but they don't seem to conflict. What am I missing?

Reply:
Snake case requires no capital letters. The regex explicitly includes them.

Reply:
Aha, yes. Thanks for the clarification.

Reply:
OK, removed the bullet with the regex.